src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+/**
+ * Calculates the rental cost based on the number of days rented,
+ * applying different discounts based on the duration.
+ *
+ * @param {number} days - The number of days the item is rented for.
+ * @return {number} The total rental cost.
+ */
 function calculateRentalCost(days) {
```

Comment:
The function name 'calculateRentalCost' is descriptive and clearly indicates its purpose.